Source/Fuse.Scripting.JavaScript/JavaScript.Dependencies.uno

```diff
@@ -133,8 +133,12 @@ namespace Fuse.Reactive
 			_moduleInstance = new Fuse.Scripting.JavaScript.ModuleInstance(Worker, this);
 		}
 
-		void DisposeModuleInstance()
+		protected bool _preserveModuleInstance = false;
+		protected void DisposeModuleInstance()
 		{
+			if (_preserveModuleInstance)
+				return;
+				
```

Comment:
This here isn't exactly a beautiful work of art. Calling DisposeModuleInstance() when we really don't want to do it seems counter-intuitive.

I have a feeling that the right thing would have been to avoid calling DisposeModuleInstance() in this case 
in `JavaScript.OnUnrooted()`. I doubt `JavaScript.DispatchEvaluate()` wants this to be preserved anyway...

Reply:
By the way, I don't think the above is a deal-breaker, just a notice. It'd be nice to clean up things like this.

Reply:
Hmm, I think I agree. Only the one call-site should preserve it, since the other one happily overwrites it anyway (but I don't think that's called if the is a preserved instance, thus this broken code doesn't fail).